chart/templates/deployment.yaml

```diff
@@ -181,3 +186,17 @@ spec:
         emptyDir: {}
   {{- end }}
 {{- end }}
+{{- if and .Values.customLogos.enabled (or (eq .Values.customLogos.volumeKind "persistentVolumeClaim") (and (eq .Values.customLogos.volumeKind "configMap") (.Values.customLogos.volumeName))) }}
+      - name: custom-logos
+  {{- if (eq .Values.customLogos.volumeKind "persistentVolumeClaim") }}
+        persistentVolumeClaim:
+          claimName: {{ if not .Values.customLogos.volumeName -}}
+      {{ template "rancher.fullname" . }}-custom-logos
+    {{- else -}}
+      {{ .Values.customLogos.volumeName }}
+    {{- end -}}
```

Comment:
Should be able to use default for these lines https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function

Reply:
done